blade-graphics/src/gles/command.rs

```diff
@@ -806,6 +810,65 @@ impl super::Command {
                     None,
                 );
             }
+
+            Self::BlitFramebuffer { from, to } => {
+                /*
+                    TODO(ErikWDev): Validate
```

Comment:
Please follow these instructions - https://github.com/kvark/blade/tree/main/blade-graphics#opengl-es
and let me know if something doesn't work!

Reply:
Sadly, I am unable to get it to work.

```yaml
[2024-12-07T20:55:05Z ERROR blade_graphics::hal::platform] EGL 'eglCreatePlatformWindowSurface' code 0x300b: _eglCreateWindowSurfaceCommon
thread 'main' panicked at blade-graphics/src/gles/egl.rs:431:26:
called `Result::unwrap()` on an `Err` value: BadNativeWindow
```

I have tried forcing X11 and Wayland and both return the same issue. I am running on  Debian GNU/Linux 12 (bookworm) x86_64 with kernel 6.1.0-28-amd64

Reply:
Ok, good to know, thank you for trying!
This has probably regressed recently due to #203
I'll check on a few more devices I have...